app/src/main/java/pl/droidsonroids/droidsmap/base/BaseFirebaseStorageInteractor.kt

```diff
@@ -0,0 +1,11 @@
+package pl.droidsonroids.droidsmap.base
+
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+
+abstract class BaseFirebaseStorageInteractor {
+    protected val firebaseStorage: FirebaseStorage = FirebaseStorage.getInstance()
+    protected lateinit var storageQueryNode: StorageReference
+
+    abstract fun setStorageNode()
```

Comment:
Hmm to nie jest taki setter w bazowym pojęciu. Każdy interactor wie, na jaki node ustawić referencję do bazy, dlatego nie ma argumentu